lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -1,4 +1,18 @@
 <div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+
+    <div class="requirements">
+        <h3 class="title"><%- gettext( "What You Need for Verification" ) %></h3>
+        <div class="requirement">
+            <h4 class="title"><%- gettext( "Webcam" ) %></h4>
+            <p class="copy"><%- gettext( "Make sure you have a webcam on your computer and allow camera access when the browser asks." ) %></p>
+        </div>
+        <div class="requirement">
+            <div class="title"><%- gettext( "Photo Identification" ) %></div>
+            <p class="copy"><%- gettext( "You will need a driver's license, passport, or government-issued ID with your name and photo." ) %></p>
```

Comment:
You need a driver's license, passport, or other government-issued ID that has your name and photo.